client/src/components/MobilePaginationDotsContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FaCircle } from 'react-icons/fa';
+import '../../dist/styles.css';
+
+const MobilePaginationDotsContainer = (props) => {
+  let dots = props.photo.map((photo, i) => {
+    const photoClassName = i === props.currentPhoto ? "activeDot" : "dot";
+      return <div key={i} className={photoClassName}><FaCircle /></div>
+  });
+
+  return (
+    <React.Fragment>
```

Comment:
This fragment is unnecessary here. You don't have two siblings being rendered next to each other.